Panic on arm running swarm tests

Arm CI builds are failing https://jenkins.dockerproject.org/job/Docker%20Master%20(arm)/3771/console

Daemon logs for

```
04:21:59 ----------------------------------------------------------------------
04:21:59 FAIL: docker_api_service_update_test.go:20: DockerSwarmSuite.TestAPIServiceUpdatePort
04:21:59 
04:21:59 [d7e6d70e8a60c] waiting for daemon to start
04:21:59 [d7e6d70e8a60c] daemon started
04:21:59 docker_api_service_update_test.go:26:
04:21:59     waitAndAssert(c, defaultReconciliationTimeout, d.checkActiveContainerCount, checker.Equals, 1)
04:21:59 daemon.go:551:
04:21:59     c.Assert(err, checker.IsNil)
04:21:59 ... value *exec.ExitError = &exec.ExitError{ProcessState:(*os.ProcessState)(0x12d751d0), Stderr:[]uint8(nil)} ("exit status 1")
04:21:59 
04:22:07 
```
https://gist.github.com/tonistiigi/c566c87c3f8b2a9511486ebe6a83784a

```
time="2016-11-11T04:21:57.982077103Z" level=debug msg="Root CA updated successfully" cluster.id=wap60eb167wyot2e7kkoa9k2a method="(*Server).updateCluster" module=ca 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x4 pc=0x363cac]

goroutine 121 [running]:
panic(0x121adf8, 0x11e28008)
	/usr/local/go/src/runtime/panic.go:500 +0x33c
sync/atomic.addUint64(0x1259d45c, 0x1, 0x0, 0x1, 0x0)
	/usr/local/go/src/sync/atomic/64bit_arm.go:31 +0x68
github.com/docker/docker/vendor/github.com/hashicorp/serf/serf.(*LamportClock).Increment(0x1259d45c, 0x45ab18, 0x121bcce0)
	/go/src/github.com/docker/docker/vendor/github.com/hashicorp/serf/serf/lamport.go:24 +0x40
github.com/docker/docker/vendor/github.com/docker/libnetwork/networkdb.(*NetworkDB).JoinNetwork(0x1259d420, 0x12313e00, 0x19, 0x0, 0x0)
	/go/src/github.com/docker/docker/vendor/github.com/docker/libnetwork/networkdb/networkdb.go:413 +0x4c
github.com/docker/docker/vendor/github.com/docker/libnetwork.(*network).joinCluster(0x123eea90, 0x0, 0x0)
	/go/src/github.com/docker/docker/vendor/github.com/docker/libnetwork/agent.go:372 +0x90
github.com/docker/docker/vendor/github.com/docker/libnetwork.glob..func1(0x1af7508, 0x123eea90, 0x123eea90)
	/go/src/github.com/docker/docker/vendor/github.com/docker/libnetwork/controller.go:725 +0x50
github.com/docker/docker/vendor/github.com/docker/libnetwork.(*controller).NewNetwork(0x1207fc80, 0x1291d440, 0x7, 0x1291d3b9, 0x7, 0x12313e00, 0x19, 0x1232e7e0, 0x7, 0x8, ...)
	/go/src/github.com/docker/docker/vendor/github.com/docker/libnetwork/controller.go:715 +0x984
github.com/docker/docker/daemon.(*Daemon).createNetwork(0x12074780, 0x1, 0x1291d440, 0x7, 0x0, 0x12331980, 0x0, 0x12331920, 0x0, 0x1291d3b9, ...)
	/go/src/github.com/docker/docker/daemon/network.go:295 +0xc6c
github.com/docker/docker/daemon.(*Daemon).SetupIngress.func1(0x12074780, 0x12313e00, 0x19, 0x1, 0x1291d440, 0x7, 0x0, 0x12331980, 0x0, 0x12331920, ...)
	/go/src/github.com/docker/docker/daemon/network.go:149 +0x150
created by github.com/docker/docker/daemon.(*Daemon).SetupIngress
	/go/src/github.com/docker/docker/daemon/network.go:187 +0xa0

goroutine 1 [chan receive]:
```

Hard to say what is the first commit with this problem because we have limited history but I can bisect it out if needed.

@mrjana @mavenugo

--- a/vendor.conf
+++ b/vendor.conf
@@ -23,7 +23,7 @@ github.com/RackSec/srslog 365bf33cd9acc21ae1c355209865f17228ca534e
 github.com/imdario/mergo 0.2.1
 
 #get libnetwork packages
-github.com/docker/libnetwork 1861587d0fe7cdf85b89160ed36f20b81e96528d
+github.com/docker/libnetwork 57be722e077059d1ee0539be31743a3642ccbeb3
 github.com/docker/go-events 18b43f1bc85d9cdd42c05a6cd2d444c7a200a894
 github.com/armon/go-radix e39d623f12e8e41c7b5529e9a9dd67a1e2261f80
 github.com/armon/go-metrics eb0af217e5e9747e41dd5303755356b62d28e3ec

